include/seeds.proposals.hpp

```diff
@@ -242,6 +251,16 @@ CONTRACT proposals : public contract {
 
         uint64_t primary_key()const { return account.value; }
       };
+
+      TABLE delegate_trust_table { // scoped by proposal's category (alliance, campaign, etc)
+        name delegator;
+        name delegatee;
+        double weight;
```

Comment:
cool!

Reply:
jaja thanks, but I am not using the weight yet, I wasn't sure where I had to use it.

Reply:
It's cool weight is in there